src/inline_table.rs

```diff
@@ -309,6 +309,9 @@ impl TableLike for InlineTable {
     fn get_mut<'s>(&'s mut self, key: &str) -> Option<&'s mut Item> {
         self.items.get_mut(key).map(|kv| &mut kv.value)
     }
+    fn get_values<'s, 'c>(&'s self, children: &'c mut Vec<(Vec<&'s Key>, &'s Value)>) {
```

Comment:
is children the return type? Not sure I understand the API

Reply:
Yes, `children` is an out-param.  This saved us from having to create a ton of small vecs that get merged together when recursing.

I guess that might not actually be needed since we have separate entry point vs recursing.  As part of this, I'll go clean that up.

Reply:
CI is green, I'll just do it in another PR